public/js/app/pages/CreateNewMemo.jsx

```diff
@@ -377,18 +395,21 @@ function CreateNewMemo(props) {
                         data-testid="form-choose-new"
                         className="form-check"
                         id="choose-from-rounds-list"
-                        key={'new' + round.ladokRoundId}
+                        key={'new' + round.applicationCodes[0]}
```

Comment:
what happens if there are not any code in applicationCodes?

Reply:
Changed array to direct string. Now round.applicationCode will be used.